django/contrib/admin/static/admin/css/responsive_rtl.css

```diff
@@ -69,7 +69,8 @@
         margin-right: 15px;
     }
 
-    [dir="rtl"] .aligned ul {
+    [dir="rtl"] .aligned ul,
+    [dir="rtl"] form .aligned ul.errorlist {
```

Comment:
I would have thought `[dir="rtl"] .aligned ul` should capture `[dir="rtl"] form .aligned ul.errorlist`. Does it work without this change?

Reply:
It works without this PR, however it's now required as we've added:
```css
form .wide ul.errorlist {
    margin-right: 200px;
    margin-left: 0px;
}
```

This is the same we did for LTR languages in 39d1e45227e060746ed461fddde80fa2b6cf0dcd.